Add plotting notebook and web scrape test

crforrester opened the pull request:
Here are some plots I made using the data from TESS, in addition to some code that can download the csv from the exofop website.

crforrester committed:
Add files via upload

arjunsavel commented:
Great work! Based on our conversation, I think it might be good to change your web-scraping notebook to a .py file so that it can be better integrated with the others' code.

crforrester committed:
Delete Web Scrape Test Run.ipynb

crforrester committed:
Delete ExoFop Plots.ipynb

crforrester committed:
Add files via upload

arjunsavel reviewed (changes requested):
Great work! Try to change the ExoFop Plots file to include a function that allows users to input the parameters they'd like to plot. For example, if they call ```plot('radius', 'period')```, it would spit out a plot of the ExoFop data along these parameters.

crforrester committed:
Add files via upload

Filters the existing TESS data to find all TOI's that have an insolation flux comparable to that of the earth's. It also applies a boundary condition of 0.5-2.5 earth radii in order to also ensure said TOI's are more than likely terrestrial.

arjunsavel commented:
Looks nice! Last thing to do with this bit is to solidify the HZ boundaries based on Hsu et al. 2020.

crforrester committed:
Delete Finding Habitable TOI's Using Insolation Flux.py

crforrester committed:
Add files via upload

Shows different lists of potentially habitable planets using different types of parameters including insolation flux and parameters as defined by Zink and Hansen (2019) and Hsu et al (2020).

arjunsavel reviewed (commented):
Thanks for making the update! One more thing — I think you might've used `P_Rad` to cut on the exofop table for the Zink + Hansen criterion (beginning line 27) before defining P_Rad (line 30). Other than that, looks good to me!

crforrester committed:
Update Finding Habitable TOI's.py

Made some corrections to the file where variables such as P_rad were being used before they defined

crforrester committed:
Update Finding Habitable TOI's.py

crforrester committed:
Add files via upload

This file computes TOI RV semi-amplitudes and plots them on a histogram
